internal/services/network/web_application_firewall_policy_resource.go

```diff
@@ -195,6 +195,47 @@ func resourceWebApplicationFirewallPolicy() *pluginsdk.Resource {
 											string(network.OwaspCrsExclusionEntrySelectorMatchOperatorStartsWith),
 										}, false),
 									},
+									"exclusion_managed_rule_set": {
```

Comment:
this is in a block called `managed_rules`, so should it maybe be
```suggestion
									"excluded_rule_set": {
```

so we get `managed_rules.excluded_rule_set`